src/pages/onboarding/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="theme-color" content="#00AEF0" />
+    <title>Welcome to Ghostery</title>
```

Comment:
I put that title here, but I am not sure if we want something else, as we don't translate it.